index.md

```diff
@@ -20,34 +20,9 @@ The toolbx environment is based on an [OCI](https://www.opencontainers.org/) ima
 
 See our guides on [installing & getting started](install) with Toolbx and [Linux distro support](distros).
 
-## Articles & Blog Posts
-#### 2022
-* [Video talk — Toolbx: developing applications on OSTree based OSes](https://www.youtube.com/watch?v=HxM15UOVmyA&t=20575s) by Debarshi Ray
-
-#### 2021
-* [Logging to registries with Toolbox](https://harrymichal.undo.it/posts/2021/logging-to-registries-with-toolbox/) by Ondřej Míchal
-* [Toolbx, a developer's new best friend!](https://fedoramagazine.org/toolbx-a-developers-new-best-friend/) by Fedora Magazine
-* [We are now on Matrix](https://debarshiray.wordpress.com/2021/11/24/toolbx-is-now-on-matrix/) by Debarshi Ray
-* [Toolbx: Redhat is hiring a software engineer](https://debarshiray.wordpress.com/2021/11/15/toolbx-red-hat-is-hiring-a-software-engineer/) by Debarshi Ray
-* [Toolbox is now Toolbx](https://debarshiray.wordpress.com/2021/11/10/toolbox-is-now-toolbx/) by Debarshi Ray
-* [Toolbox — After a gap of 15 months](https://debarshiray.wordpress.com/2021/01/14/toolbox-after-a-gap-of-15-months/) by Debarshi Ray
-* [Video talk — Interactive container environment](https://www.youtube.com/watch?v=qdpg-zBvNz8) by Ondřej Míchal
-
-#### 2020
-* [Public lightweight sway developer desktop with OSTree and podman/toolbox](https://piware.de/post/2020-12-13-ostree-sway/) by Martin Pitt
-* [Toolbox Status Update](https://harrymichal.undo.it/posts/2020/toolbox-status-update-v0.0.92-v0.0.96/) by Ondřej Míchal
-* [A little collection of ‘How to do X with Toolbox on Fedora Silverblue’](https://harrymichal.undo.it/posts/2020/a-little-collection-of-how-to-do-x-with-toolbox-on-fedora-silverblue/) by Ondřej Míchal
-* [A quick guide to podman and toolbox in Debian (and maybe Ubuntu)](https://dev.to/bureado/a-quick-guide-to-podman-and-toolbox-in-debian-5672) by José Miguel Parrella
-
-#### 2019
-* [A quick introduction to Toolbox](https://fedoramagazine.org/a-quick-introduction-to-toolbox-on-fedora/) by Fedora Magazine
-* [Toolbox — A fall 2019 update](https://debarshiray.wordpress.com/2019/11/01/toolbox-a-fall-2019-update/) by Debarshi Ray
-* [Fedora Toolbox is now just Toolbox](https://debarshiray.wordpress.com/2019/04/03/fedora-toolbox-is-now-just-toolbox/) by Debarshi Ray
-* [Fedora Toolbox — Under the hood](https://debarshiray.wordpress.com/2019/01/21/fedora-toolbox-under-the-hood/) by Debarshi Ray
-* [Video talk — Toolbox: using Silverblue for development](https://www.youtube.com/watch?v=BGXs0W6NRBM) by Debarshi Ray
-
-#### 2018
-* [Fedora Toolbox — Hacking on Fedora Silverblue](https://debarshiray.wordpress.com/2018/10/22/fedora-toolbox-hacking-on-fedora-silverblue/) by Debarshi Ray
+## Articles, Talks & Blog Posts
+
+See our [articles, talks & blog posts](articles) .
```

Comment:
I wonder if we can fold this into some other part of the top-level landing page.  Such a short section that's just a link seems a bit redundant.

Maybe this could be moved to the *Installation & Use* section?  A big chunk of the articles and talks, if not all of them, are about how Toolbx can be used.  So, after giving the reader a very brief introduction on how to install and get started, and what Linux distributions are supported, we can point them to the articles and talks for further exploration.